packages/jest-runtime/src/index.js

```diff
@@ -445,20 +445,22 @@ class Runtime {
     this._mockRegistry = Object.create(null);
     this._moduleRegistry = Object.create(null);
 
-    if (this._environment && this._environment.global) {
-      const envGlobal = this._environment.global;
-      Object.keys(envGlobal).forEach(key => {
-        const globalMock = envGlobal[key];
-        if (
-          (typeof globalMock === 'object' && globalMock !== null) ||
-          typeof globalMock === 'function'
-        ) {
-          globalMock._isMockFunction === true && globalMock.mockClear();
-        }
-      });
+    if (this._environment) {
+      if (this._environment.global) {
+        const envGlobal = this._environment.global;
+        Object.keys(envGlobal).forEach(key => {
+          const globalMock = envGlobal[key];
+          if (
+            (typeof globalMock === 'object' && globalMock !== null) ||
+            typeof globalMock === 'function'
+          ) {
+            globalMock._isMockFunction === true && globalMock.mockClear();
+          }
+        });
+      }
 
-      if (envGlobal.mockClearTimers) {
-        envGlobal.mockClearTimers();
+      if (this._environment.fakeTimers) {
+        this._environment.fakeTimers.clearAllTimers();
```

Comment:
is this expected?

Reply:
This behavior is there from the very first commit: https://github.com/facebook/jest/commit/88a94d5d1bc1f387317a3068bf510ab992c5dc64#diff-c57162543a4f0bc311cba66ba52baf7bR936

BTW, look what's 5 yrs deprecated too: https://github.com/facebook/jest/commit/88a94d5d1bc1f387317a3068bf510ab992c5dc64#diff-b689d4f4c53abcb13c3c1d211b4d8bc5R55 😄

Reply:
haha, "very soon" indeed